config/api/routes/files.php

```diff
@@ -1,12 +1,15 @@
 <?php
 
+// routing pattern to match all models with files
+$pattern = '(site|users/.*?|pages/.*?)';
```

Comment:
I wasn't able to dive deeper into this, but as far as I understand it, user and page URIs never contain slashes (instead the `+` sign). If we want to make sure, we could use the following pattern:

```suggestion
$pattern = '(site|users/[^/]+|pages/[^/]+)';
```

The `+` instead of the `*` also makes sure that the user/page URI cannot be empty. The question mark at the end is not needed when the character group cannot include slashes.

I also don't really get why this PR fixes the issue as even `.*/files` would only ever match if `/files` follows after the page URI. Very weird.